patterns/testimonials-two-columns-avatar.php

```diff
@@ -0,0 +1,61 @@
+<?php
+/**
+ * Title: Two columns with avatar
+ * Slug: twentytwentyfive/testimonials-two-columns-avatar
+ * Keywords: testimonial
+ * Categories: testimonials
+ *
+ * @package WordPress
+ * @subpackage Twenty_Twenty_Five
+ * @since Twenty Twenty-Five 1.0
+ */
+
+?>
+
+<!-- wp:group {"align":"wide","style":{"spacing":{"padding":{"top":"var:preset|spacing|60","bottom":"var:preset|spacing|60"}}},"layout":{"type":"constrained"}} -->
```

Comment:
This group should be set to full, so that if we add a background color it looks good.

![Captura de ecrã 2024-08-30, às 16 58 05](https://github.com/user-attachments/assets/b71df2c8-d36d-426a-a1b7-51e4fa0af528)

Reply:
Thanks, updated in https://github.com/WordPress/twentytwentyfive/pull/118/commits/1c58d688ee227007220d325ba4b250926a49eb9f